Tests/UnitTests/MessageBirdUnitTests/MockRestClient.cs

```diff
@@ -104,10 +109,11 @@ public MockRestClient AndReturns(string responseBody)
         /// <summary>
         /// Sets the expected (HTTP) method.
         /// </summary>
-        public MockRestClient FromEndpoint(string method, string uri)
+        public MockRestClient FromEndpoint(string method, string uri, string endpoint = null)
```

Comment:
What do you think about renaming `endpoint`? I think it's not very obvious what the difference between `uri` and `endpoint` is. Perhaps `baseUri`/`baseUrl`?

Reply:
yes